server/ntb/scanpix/scanpix_datalayer.py

```diff
@@ -35,6 +35,7 @@
     'viewImage': ('preview', 'thumbnail_big', 'thumbnail', 'preview_big'),
     'baseImage': ('mp4_preview', 'mp4_thumbnail', 'preview_big', 'preview', 'thumbnail_big', 'thumbnail')}
 logger = logging.getLogger('ntb:scanpix')
+SCANPIX_TZ = 'Europe/Oslo'
```

Comment:
Yeah, actually good idea, can add to settings.py @petrjasek does that make sense?